operator/doublewrite/metrics.go

```diff
@@ -10,46 +10,46 @@ import (
 
 func NewMetrics() *Metrics {
 	return &Metrics{
-		IdentityCRDTotalCount: metric.NewGauge(metric.GaugeOpts{
+		IdentityCRDTotal: metric.NewGauge(metric.GaugeOpts{
 			Namespace: metrics.CiliumOperatorNamespace,
-			Name:      "doublewrite_identity_crd_total_count",
+			Name:      "doublewrite_identity_crd_total",
 			Help:      "The total number of CRD identities (Requires the Double-Write Identity allocation mode to be enabled)",
 		}),
 
-		IdentityKVStoreTotalCount: metric.NewGauge(metric.GaugeOpts{
+		IdentityKVStoreTotal: metric.NewGauge(metric.GaugeOpts{
 			Namespace: metrics.CiliumOperatorNamespace,
-			Name:      "doublewrite_identity_kvstore_total_count",
+			Name:      "doublewrite_identity_kvstore_total",
 			Help:      "The total number of identities in the KVStore (Requires the Double-Write Identity allocation mode to be enabled)",
 		}),
 
-		IdentityCRDOnlyCount: metric.NewGauge(metric.GaugeOpts{
+		IdentityCRDOnly: metric.NewGauge(metric.GaugeOpts{
 			Namespace: metrics.CiliumOperatorNamespace,
-			Name:      "doublewrite_identity_crd_only_count",
+			Name:      "doublewrite_identity_crd_only",
 			Help:      "The number of CRD identities not present in the KVStore (Requires the Double-Write Identity allocation mode to be enabled)",
 		}),
 
-		IdentityKVStoreOnlyCount: metric.NewGauge(metric.GaugeOpts{
+		IdentityKVStoreOnly: metric.NewGauge(metric.GaugeOpts{
 			Namespace: metrics.CiliumOperatorNamespace,
-			Name:      "doublewrite_identity_kvstore_only_count",
+			Name:      "doublewrite_identity_kvstore_only",
```

Comment:
Typically metrics should end with the unit. I don't think `only` is a unit, maybe this should be `doublewrite_identity_kvstore_only_total`? (Same for `doublewrite_identity_crd_only`)

Reply:
You are right that the metrics look a bit weird now. I added the `_total` suffix, hope it's better